tests/unit/src/main/java/org/mule/tck/util/CompilerUtils.java

```diff
@@ -15,6 +15,14 @@
 import static org.mule.runtime.api.util.Preconditions.checkArgument;
 import static org.mule.tck.ZipUtils.compress;
 
+import org.mule.runtime.core.api.util.ClassUtils;
+import org.mule.runtime.core.api.util.StringUtils;
+import org.mule.tck.ZipUtils;
+
+import org.apache.commons.io.filefilter.NameFileFilter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import java.io.File;
```

Comment:
@elrodro83 same as in https://github.com/mulesoft/mule/pull/7786/files#r273057541